components/TProfile.vue

```diff
@@ -137,12 +155,12 @@
     <TProfileContacts :profile="profile" class="p-2 mb-4 bg-gray-100" />
 
     <div v-if="uid === profile.id" class="flex justify-center space-x-2">
-      <TButton label="Edit Profile" to="/settings?tab=profile" />
+      <TButton :label="$t('myprofile.edit')" to="/settings?tab=profile" />
     </div>
 
     <WTeaser
       v-if="profile.partner === 'Yes'"
-      title="I am looking for a dance partner"
+      :title="$t('myprofile.Wteaser')"
```

Comment:
That's not just in "My Profile". It's shown on any profile. The `TProfile` component renders any profile.
"My Profile" makes sense if it's inside the `v-if="uid === profile.id"`.

```suggestion
      :title="$t('myprofile.Wteaser')"
```